### PR TITLE
Issue #5338

### PR DESCRIPTION
**Pull request by sanketio**
Fix - Undefined index `id` for dashboard widget #5338

### CODE REVIEW

File: includes/admin/dashboard-widgets.php [hunk 1]
```diff
@@ -175,7 +175,7 @@ function edd_load_dashboard_sales_widget( ) {
 									<?php echo get_the_title( $payment->ID ) ?>
 									&mdash; <?php echo $payment->user_info['email'] ?>
 								</a>
-								<?php if ( $payment->user_info['id'] > 0 ) {
+								<?php if ( ! empty( $payment->user_info['id'] ) && ( $payment->user_info['id'] > 0 ) ) {
```
Comment (pippinsplugins) [inline includes/admin/dashboard-widgets.php]: This fix works but let's go a step further and use `$payment->user_id` instead. That helps move us away from the `user_info` array and it's value will always be set.

Comment (sanketio) [inline includes/admin/dashboard-widgets.php]: @pippinsplugins 
Should I replace for every instance in that portion or just the condition?

Comment (pippinsplugins) [inline includes/admin/dashboard-widgets.php]: Just this one.

File: includes/admin/dashboard-widgets.php [hunk 2]
```diff
@@ -175,7 +175,7 @@ function edd_load_dashboard_sales_widget( ) {
 									<?php echo get_the_title( $payment->ID ) ?>
 									&mdash; <?php echo $payment->user_info['email'] ?>
 								</a>
-								<?php if ( ! empty( $payment->user_info['id'] ) && ( $payment->user_info['id'] > 0 ) ) {
+								<?php if ( ! empty( $payment->user_id ) && ( $payment->user_info['id'] > 0 ) ) {
```
Comment (pippinsplugins) [inline includes/admin/dashboard-widgets.php]: `&& ( $payment->user_info['id'] > 0 )` needs to be adjusted too.

**Comment by cklosowski** [comment]
This PR has a dirty commit history. I'm going to make a new one, preserving the commit history of @sanketio so it's a cleaner merge changeset

**Pull request closed without merging**